src/full/Agda/Termination/TypeBased/Graph.hs

```diff
@@ -146,15 +147,15 @@ instantiateComponents baseSize rigids clusterMapping graph subst (comp : is) = d
            -- Some element in the component has is bigger or equal than infinity,
            -- which means that the component should be assigned to infinity
            baseSize
-        | head comp `IntSet.member` bottomVars =
+        | headWithDefault __IMPOSSIBLE__ comp `IntSet.member` bottomVars =
```

Comment:
`comp` could be a `List1` if the following PR lands:
- https://github.com/agda/agda/pull/7079